components/shared/CopyContainer.tsx

```diff
@@ -0,0 +1,49 @@
+import { CopyIcon } from "@primer/octicons-react";
+import { useRef, useState } from "react";
+import { Options } from "html2canvas";
+import { Spinner } from "components/atoms/SpinLoader/spin-loader";
+import { useToast } from "lib/hooks/useToast";
+import { copyNodeAsImage } from "lib/utils/copy-to-clipboard";
+
+interface CopyContainerProps {
+  copySuccessMessage?: string;
+  copyErrorMessage?: string;
+  options?: Partial<Options>;
```

Comment:
The type will provide Intellisense, but here's the docs https://html2canvas.hertzen.com/configuration